src/main/java/edu/utdallas/davisbase/parser/Parser.java

```diff
@@ -15,13 +34,204 @@ public Parser(ParserConfiguration configuration) {
+          createIndexStatement.toString(),
+          createIndexStatement.getTable().getName(),
+          createIndexStatement.getIndex().getName(),
+          createIndexStatement.getIndex().getColumnsNames().get(0)
+        );
+        return createIndex;
+      }
+
+      else if (stmt instanceof Insert) {
+        Insert insertStatement = (Insert) stmt;
+        List<String> cols = new ArrayList<>();
+        for(Column c: insertStatement.getColumns()){
+          cols.add(c.getColumnName());
+        }
+        String vals = insertStatement.getItemsList().toString().replaceAll("[('')]", "");
+        String[] splitValues=vals.trim().split("\\s*,\\s*");
+        InsertCommandRepresentation insert = new InsertCommandRepresentation(
+          insertStatement.toString(),
+          insertStatement.getTable().getName(),
+          cols,
+          Arrays.asList(splitValues)
+        );
+        return insert;
+      }
+      if (stmt instanceof Delete) {
+        Delete deleteStatement = (Delete) stmt;
+        DeleteCommandRepresentation delete = new DeleteCommandRepresentation(
+          deleteStatement.toString(),
+          deleteStatement.getTable().getName(),
+          parseWhereExpression(deleteStatement.getWhere())
+        );
+        return delete;
+      }
+      else if (stmt instanceof Update) {
+        Update updateStatement = (Update) stmt;
+        List<String> cols = new ArrayList<>();
+        for(Column c: updateStatement.getColumns()){
+          cols.add(c.getColumnName());
+        }
+        String vals = updateStatement.getExpressions().toString().replaceAll("[\\[''\\]]", "");
+        String[] splitValues=vals.trim().split("\\s*,\\s*");
+        UpdateCommandRepresentation update = new UpdateCommandRepresentation(
+          updateStatement.toString(),
+          updateStatement.getTables().get(0).getName(),
+          cols,
+          Arrays.asList(splitValues),
+          parseWhereExpression(updateStatement.getWhere())
+        );
+        return update;
+      }
+      else if(stmt instanceof Select){
+        Select selectStatement = (Select) stmt;
+        String selectBody = selectStatement.getSelectBody().toString();
+        String[] splitStr = selectBody.trim().split("SELECT |FROM ");
+        String[] splitValues=splitStr[1].trim().split("\\s*,\\s*");
+        Expression exp;
+        for(String col:splitValues){
+            try {
+              exp = CCJSqlParserUtil.parseExpression(col);
+              if(!(exp instanceof Column)){
+                throw new ParseException(exp + "is not a supported expression in Davisbase");
+              }
+            }catch(JSQLParserException e){
+              if(!col.equalsIgnoreCase("*")){
+                throw new ParseException(e);
+              }
+          }
+        }
+        SelectCommandRepresentation select = new SelectCommandRepresentation(
+          selectStatement.toString(),
+          splitStr[2],
+          Arrays.asList(splitValues),
```

Comment:
Same comments as above. This should be a structured list of JSqlParser literal value expression objects. Otherwise, the compiler will just have to parse the strings into their respective literal value types all over again.